docs/reference/how-tos.md

```diff
@@ -0,0 +1,221 @@
+# How-tos
+
+## How to get my Android application instance [get-application]
+
+Your [Application](https://developer.android.com/reference/android/app/Application) instance is needed to initialize the agent. There are a couple of ways you can get yours:
+
+### From within your custom Application implementation (recommended)
+
+Ideally, the agent should get initialized as soon as your application is launched, to make sure that it can start collecting telemetry from the very beginning.
```

Comment:
```suggestion
Ideally, the agent should get initialized as soon as your application is launched to make sure that it can start collecting telemetry from the very beginning.
```